README.adoc

```diff
@@ -334,18 +334,19 @@ end
 
 # good
 describe '#attributes' do
-  subject { FactoryBot.create(:article) }
+  let(:article) { FactoryBot.create(:article) }
+  subject { article.attributes }
 
   specify do
-    expect(subject.attributes).to include subject.display_name
-    expect(subject.attributes).to include subject.created_at
+    expect(subject).to include(display_name: article.display_name)
+    expect(subject).to include article.created_at
   end
 end
```

Comment:
This whole block is a leftover, buggy, obscure and off the point, that I forgot to remove.
A good example is below.

Appreciate if you remove it in a separate commit.

Reply:
I didn't quite get what you meant by this - I hope the changes I've made here are sufficient.